airflow/dags/scrape_state_geoportal/state_highway_network.yml

```diff
@@ -0,0 +1,10 @@
+operator: operators.StateGeoportalAPIOperator
+
+root_url: 'https://caltrans-gis.dot.ca.gov/arcgis/rest/services/'
+service: "CHhighway/SHN_Lines"
+layer: "0"
+product: 'state_highway_network'
+where: "1=1"  # You can change this to filter data
+outFields: "*"  # Specify the fields to return
+f: "geojson"  # Format of the response
```

Comment:
**suggestion:** These are no longer necessary with the new defaults, right? Should probably remove them (but verify that it still works with them removed).

Reply:
removed and still ran successfully!